### PR TITLE
Feature/tasks

### PR DESCRIPTION
**Pull request by moklidia**

### CODE REVIEW

File: js-routes/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Contributor Covenant Code of Conduct
```
Comment (artempartos) [inline js-routes/CODE_OF_CONDUCT.md]: why you have gem js routes in the repo?)

File: app/javascript/packs/components/TaskBoard.js [hunk 2]
```diff
@@ -1,28 +1,175 @@
...
+  }
 
-export default class TaskBoard extends React.Component {
   render() {
-    return <Board data={data} />
+    return <div>
+      <h1>Your tasks</h1>
+      <Button variant="primary" onClick={this.handleAddShow}>Create new task</Button>
+      <Board
+        data={this.getBoard()}
```
Comment (artempartos) [inline app/javascript/packs/components/TaskBoard.js]: components={components}

File: app/javascript/packs/components/TaskBoard.js [hunk 3]
```diff
@@ -1,28 +1,175 @@
...
+        break;
+    }
+  }
+
+  handleEditShow = () => {
+    this.setState({ editPopupShow: true });
+  }
 
-export default class TaskBoard extends React.Component {
   render() {
```
Comment (artempartos) [inline app/javascript/packs/components/TaskBoard.js]: const components = {
  LaneHeader,
};

**Comment by artempartos** [comment]
Also, it' better to setup webpack dev server in development env

**Comment by artempartos** [comment]
Because it' too long to wait )

**Comment by artempartos** [comment]
Try gem foreman 

Procfile
```
web:     bundle exec rails s -b '0.0.0.0' -p 3000
webpack: bin/webpack-dev-server --watch
```

docker-compose.yml
```
services:
  web:
    ...
    command: bundle exec foreman start
```

File: app/assets/stylesheets/bootstrap_and_overrides.css.less [hunk 4]
```diff
@@ -19,3 +19,10 @@
 //
 // Example:
 // @link-color: #ff0000;
+
+.modal-backdrop.show {
+    opacity: 0.8;
+}
+.fade {
+	opacity: 1.0;
```
Comment (artempartos) [inline app/assets/stylesheets/bootstrap_and_overrides.css.less]: indent (

File: app/controllers/api/v1/tasks_controller.rb [hunk 5]
```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 class Api::V1::TasksController < Api::V1::ApplicationController
+  include Concerns::AuthHelper
```
Comment (artempartos) [inline app/controllers/api/v1/tasks_controller.rb]: include it on application controller level

File: app/assets/stylesheets/bootstrap_and_overrides.css.less (hunk 4 above)
Comment (moklidia) [reply app/assets/stylesheets/bootstrap_and_overrides.css.less]: Weirdly, I have the correct indent in my local repo. And when I push it, it becomes distorted again on github 
image

File: app/controllers/api/v1/tasks_controller.rb (hunk 5 above)
Comment (moklidia) [reply app/controllers/api/v1/tasks_controller.rb]: fixed

**Comment by artempartos** [comment]
Last changes
in webpacker.yml
```check_yarn_integrity: false```
To disable yarn checks new updates

in docker-compose
```
ports:
  ...
  - 3035:3035
```
To expose webpack-dev-server port on localhost

**Comment by moklidia** [comment]
@artempartos  It flies now, thanks for the tip!

File: app/assets/stylesheets/bootstrap_and_overrides.css.less (hunk 4 above)
Comment (artempartos) [reply app/assets/stylesheets/bootstrap_and_overrides.css.less]: just setup less plugin in your editor )

**Pull request merged**